### PR TITLE
btc multi sign

### PR DESCRIPTION
**Pull request by Rakeally**

### CODE REVIEW

File: android/src/main/kotlin/africa/ejara/trustdart/Coin.kt [hunk 1]
```diff
@@ -82,4 +83,19 @@ open class Coin(nameOfCoin: String, typeOfCoin: CoinType) : CoinInterface {
         return AnySigner.signJSON(opJson, privateKey.data(), coinType!!.value())
     }
 
+    override fun multiSignTransaction(
+        path: String,
```
Comment (jayluxferro) [inline android/src/main/kotlin/africa/ejara/trustdart/Coin.kt:87]: This isn't used. Please remove it if not needed.

File: android/src/main/kotlin/africa/ejara/trustdart/TrustdartPlugin.kt [hunk 2]
```diff
@@ -164,6 +164,36 @@ class TrustdartPlugin : FlutterPlugin, MethodCallHandler {
                     validator.details.errorDetails
                 )
             }
+            "multiSignTransaction" -> {
+                val coin: String? = call.argument("coin")
+                val path: String? = call.argument("path")
```
Comment (jayluxferro) [inline android/src/main/kotlin/africa/ejara/trustdart/TrustdartPlugin.kt:169]: path not needed.

File: android/src/main/kotlin/africa/ejara/trustdart/coins/BTC.kt [hunk 3]
```diff
@@ -77,4 +80,57 @@ class BTC : Coin("BTC", CoinType.BITCOIN) {
         return Numeric.toHexString(output.encoded.toByteArray())
     }
 
-}
\ No newline at end of file
+
+    override fun multiSignTransaction(
+        path: String,
```
Comment (jayluxferro) [inline android/src/main/kotlin/africa/ejara/trustdart/coins/BTC.kt:85]: path not used.

File: example/lib/operations.dart [hunk 4]
```diff
@@ -102,7 +102,13 @@ Map<String, dynamic> operations = {
     "amount": 3000,
     "fees": 1000,
     "changeAddress": "15o5bzVX58t1NRvLchBUGuHscCs1sumr2R",
-    "change": 500
+    "change": 500,
+    "privateKeys": [
+      "a321c4996143e0add05864bbb694ceb399fbe5d0884d721b1a04755f9f7497a9",
+      "bbc27228ddcb9209d7fd6f36b02f7dfa6252af40bb2f1cbc7a557da8027ff866",
```
Comment (jayluxferro) [inline example/lib/operations.dart:108]: I hope they are dummy private keys.

**Review by jayluxferro — CHANGES_REQUESTED** [review]

File: example/lib/operations.dart (hunk 4 above)
Comment (Rakeally) [reply example/lib/operations.dart:108]: Yeah

**Pull request merged**